qutebrowser/browser/commands.py

```diff
@@ -101,6 +101,11 @@ def _current_url(self):
             msg += "!"
             raise cmdexc.CommandError(msg)
 
+    def _current_title(self):
+        """Convenience method to get the current title."""
+        title = self._current_widget().title()
+        return title if title else "(null)"
```

Comment:
I'm not sure if this should be handled here - maybe rather just pass the empty string to `bookmark_add`, and handle that situation there?

Also, the user interface would actually display `(null)` now, and that'd be saved to the file - why not just not save anything at all for the title, i.e. only the URL - and handle that case when reading?